apps/web-staking/src/app/components/redeem/History.tsx

```diff
@@ -194,6 +201,10 @@ export default function History({ redemptions, reloadRedemptions }: {
 	}, [isSuccess, isError, updateOnSuccess, updateOnError]);
 
 	const onClaim = async (redemption: RedemptionRequest) => {
+		if(isApproved) {
```

Comment:
Shouldn't this only show if Not KYC Approved?

Reply:
Thats correct, this ha been updated.